src/Console/Command/CheckCommand.php

```diff
@@ -40,6 +41,7 @@ protected function configure()
                     ),
                     new InputOption('no-cache', null, InputOption::VALUE_NONE, 'Disable rule set cache'),
                     new InputOption('cache-dir', null, InputOption::VALUE_REQUIRED, 'Cache directory', '.rules/'),
+                    new InputOption('log-html', null, InputOption::VALUE_OPTIONAL, 'Log output in HTML format to file.'),
```

Comment:
@derrabus I agree. Keep in mind that options are always optional. Here I'm talking about the value of that optional option.

Right now you can do this:

```bash
$ deprecation-detector check ... --log-html
```

Result? Nothing gets generated because the `$htmlOutputPath` is the value passed to that option.

Solution?

1) Make the value of this option mandatory (`InputOption::VALUE_REQUIRED`).
2) Fix the PHP code to provide a default output path when none is set.